src/container_inspector/utils.py

```diff
@@ -116,6 +116,8 @@ def extract_tar(location, target_dir, skip_symlinks=True):
                 continue
 
             if tarinfo.islnk() or tarinfo.issym():
+                if skip_symlinks:
```

Comment:
@pombredanne The docstring says `Issue a warning if skip_symlinks is True and links target are missing.`, but I'm not sure why we would need to issue a warning if we want to skip extracting symlinks entirely? Right now, I'm skipping links but not issuing warnings.